starwars/src/App.css

```diff
@@ -1,8 +1,20 @@
 .App {
   text-align: center;
+  margin: auto;
+  max-width: 960px;
 }
 .Header {
   color: #443e3e;
   text-shadow: 1px 1px 5px #fff;
 }
 /* write your parent styles in here for your App.js */
+
+.cardOla {
+  width: 400px;
+  margin: 20px;
+}
```

Comment:
I would recommend avoiding pixels just because for a responsive website using percentages or rems is preferred.